akka-stream/src/main/scala/akka/stream/scaladsl/Tcp.scala

```diff
@@ -23,9 +24,18 @@ import scala.util.control.NoStackTrace
 object Tcp extends ExtensionId[Tcp] with ExtensionIdProvider {
 
   /**
-   * * Represents a successful TCP server binding.
+   * Represents a successful TCP server binding.
+   *
+   * @param localAddress The address the server was bound to
+   * @param unbindAction a function that will trigger unbind of the server
+   * @param whenUnbound A future that is completed when the server is unbound, or failed if the server binding fails
+   *
+   * Not indented for user construction
```

Comment:
Is putting stuff after `@param` legal for javadoc (when we gen javadoc)

Reply:
I'll move it up just to be on the sure side of things